### PR TITLE
Some small variation to the stale bot configuration file

### PR DESCRIPTION
**Pull request by ret2libc**
I have installed probot stale on Rizin repository. The configuration file was already there, I just did some small variation. Please check if you think they are ok.

### CODE REVIEW

**Review by wargio — APPROVED** [review]
💗

**Review by ITAYC0HEN — APPROVED** [review]
Alright, looks good!

File: .github/stale.yml [hunk 1]
```diff
@@ -1,21 +1,18 @@
 # Configuration for probot-stale - https://github.com/probot/stale
 
 # Number of days of inactivity before an Issue or Pull Request becomes stale
-daysUntilStale: 700
+daysUntilStale: 186
 
 # Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
 # Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
-daysUntilClose: 60
+daysUntilClose: 21
```
Comment (XVilka) [inline .github/stale.yml]: Why 21? I think marking until "stale" is fine, but lets give more time for someone when they got a notification from bot. I think 60 was a reasonable amount for a complex project such as Rizin is.

File: .github/stale.yml [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
 
 # Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
 onlyLabels: []
 
 # Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
 exemptLabels:
-  - architectures-enhancements
   - enhancement
-  - New Architecture
```
Comment (XVilka) [inline .github/stale.yml:17]: I suggest to excempt also these:
- `high-priority`
- `documentation`
- probably also `infrastructure`

**Review by XVilka — CHANGES_REQUESTED** [review]

**Review by XVilka — APPROVED** [review]

**Pull request merged**